regression/pages/enterprise/enterprise_portal_course_page.py

```diff
@@ -0,0 +1,59 @@
+"""
+Enterprise portal course start page
+"""
+from bok_choy.page_object import PageObject
+
+
+class EnterprisePortalCourse(PageObject):
```

Comment:
I would prefer generic pages with information of different portals stored in a dictionary, however if there are significant differences in flow and css between different portals we might need to create pages for each portal. I am merging this branch as well but will do something about it in next PR